pxt-cli/cli.js

```diff
@@ -70,13 +70,60 @@ function target(n) {
     console.log(`    pxt serve`)
 }
 
+function link(dir) {
+    if (!dir) {
+        console.log("No directory specified")
+        process.exit(1);
+    }
+    var absPath = path.resolve(dir);
+
+    if (!fs.existsSync(absPath)) {
+        console.log("Could not find " + absPath);
+        process.exit(1);
+    }
+
+    var modulesPath = path.resolve("node_modules");
+
+    if (!fs.existsSync(modulesPath))
+        fs.mkdirSync(modulesPath)
+
+    var corePath = path.join(modulesPath, "pxt-core");
+    if (fs.existsSync(corePath)) {
+        var stats = fs.statSync(corePath);
+        if (stats.isSymbolicLink()) {
+            fs.unlinkSync(corePath);
+        }
+        else {
+            try {
+                var rimraf = require("rimraf");
```

Comment:
The package doesn't require rimraf, but maybe this is fine

Reply:
Yeah, but pxt-core does